packages/web-components/fast-foundation/src/avatar/avatar.spec.md

```diff
@@ -0,0 +1,115 @@
+# Avatar
+
+## Overview
+
+An avatar is a graphical represention of a user or object.
+
+### Use Cases
+
+A common use case would be to display an image or text (usually initials) of a user or an object, such as in a user profile.
+
+### Features
+- A URL for an image can be passed to the component to be displayed in the coin
+  - When no image is provided, the initials of the provided name will be displayed in the coin
+- Badge slot: Able to slot in a badge component
+- Media slot: Accepts an `img` or an `svg`
+- `shape`, a circle or square shape can be chosen. Any border radius for square shaped coins should be determined by the users design system values or stylesheet
+- `color`, a hexadecimal color can be provided to determine the coin background color
+- When a `link` is provided an `aria-link` attribute is added
+
+### Prior Art/Examples
+
+- [Fluent UI (Persona)](https://developer.microsoft.com/en-us/fluentui#/controls/web/persona)
+- [Fluent UI](https://fluentsite.z22.web.core.windows.net/components/avatar/definition)
+- [Lighting Design](https://www.lightningdesignsystem.com/components/avatar/)
+- [Evergreen](https://evergreen.segment.com/components/avatar)
+- [Ant Design](https://ant.design/components/avatar/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/design-system/avatar)
+---
+### API
+
+*Component Name*
+- `fast-avatar`
+
+#### Attributes
+|   Name    | Description                                                 | Type                                |
+|-----------|-------------------------------------------------------------|-------------------------------------|
+| `name`    | Accepts name string for coin display                        | `string`                            |
+| `src` | Accepts URL string of image to be displayed                 | `string`                            |
+| `alt`| Accepts alt text for image                                  | `string`                            |
+| `link`    | Accepts a URL for the anchor source                         | `string`                            |
+| `shape`   | Determines the avatar coin shape. Default will be a circle. | `string: default | circle | square` |
+| `fill`    | Accepts a string that defines the `avatar-fill-*` post-fix for custom variable mapping.                | `string`                 |
+| `color`   | Accepts a string that defines the `avatar-color-*` post-fix for custom variable mapping.                    | `string`                |
+
+#### Slots
+
+| Name  | Description               | Elements     |
+|-------|---------------------------|--------------|
+|`badge`| Slot for fast badge       | `fast-badge` |
+|`media`| Slot for images and icons | `img`, `svg` |
+
+### Anatomy and Appearance
+
+*Template*
+```js
+<div
+    class="coin ${x => x.shape}"
+    part="coin"
```

Comment:
Where does "coin" come from? Is that a term that is being used widely in other component libraries?

I think I'd prefer a more semantic name like "backplate" to "coin", because to me it better conveys the purpose of the part.

Reply:
When I initially wrote the spec for avatar I was under the impression that the term was used in other avatars. After reviewing again, no so much.

Backplate works. I'll update.